### ISSUE TITLE
Disabling Body Temperature has no effect

### ISSUE THREAD

**Issue by mohler7154** [Glitchfiend/ToughAsNails#282]
I installed this as part of a new test modpack we are making.  After the recent update the rate of body temperature change was much faster.  While trying to learn to change the setting I set enable body temperature to false.  It had no effect.  Players were still freezing to death.

Forgive me if I missed something obvious, I am fairly new to this.

**Comment by Rexhunter99** [comment]
I'm not sure myself, 1.11.2's temperature system is certaintly not working as intended, I ran it and with biomes o plenty I was constantly dying of hyperthermia in the middle of an oak forest, I have to constantly dive int othe nearest water source to avoid dying.  There needs to be a toggle to enable/disable the entire temperature system as a whole in the Mod Options menu as I love the thirst and health mechanic but the temperature is simply overkill right now and is compeltely unmanageable.  Realistically unless you're living in Australia's forests or a desert biome, you shouldn't be constantly fighting hyperthermia, it's unreasonable.

**Comment by mohler7154** [comment]
Yeah this what I mean.  I did change the body temp flag to false and still
had the effects.  Overall I love the mod, but had to disable it in my pack
as it made gameplay unplayable.

**Comment by Rexhunter99** [comment]
Okay @mohler7154 I managed to disable it in the CFG files. I did two things. I set the following line to 'false' in the file "<minecraft-root>/config/toughasnails/temperature.cfg":
`B:"Enable Body Temperature"=false`
I then copied that line and deposited it into the "<minecraft-root>/config/toughasnails/gameplay.cfg" config file, also set to false. I believe this just lets you toggle it from the Forge Mod Options menu.
However this worked to disable it for me.  Good luck with you, I know mods are finicky.
